src/components/utils/file.js

```diff
@@ -1,16 +1,23 @@
 import loadImage from 'blueimp-load-image/js/load-image'
 import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
 
-export const fileToBase64 = (file, callback, errorCallback) => {
+export const convertFile = (file, callback, errorCallback) => {
   const reader = new FileReader();
-  reader.readAsDataURL(file);
-  reader.onload = function () {
-   callback(reader.result)
+  // If PDF return base64. For other images return blob
```

Comment:
This function needs to be split. One function converts files to base64, another one converts files to Blobs.
On the last point, I would challenge the need of converting files to Blobs, it might not be necessary, considering they are used so interchangeably.